pkg/controller/common/util.go

```diff
@@ -1212,7 +1212,7 @@ func IsErrCacheNotStarted(err error) bool {
 
 // GetDataVolumeTTLSeconds gets the current DataVolume TTL in seconds if GC is enabled, or < 0 if GC is disabled
 func GetDataVolumeTTLSeconds(config *cdiv1.CDIConfig) int32 {
-	const defaultDataVolumeTTLSeconds = 0
+	const defaultDataVolumeTTLSeconds = -1
```

Comment:
Maybe a comment here that GC is disabled by default?

Reply:
added